resources/default_pipeline_environment.yml

```diff
@@ -38,6 +38,7 @@ general:
   githubApiUrl: 'https://api.github.com'
   githubServerUrl: 'https://github.com'
   gitSshKeyCredentialsId: '' #needed to allow sshagent to run with local ssh key
+  globalExtensionsDirectory: '.pipeline/tmp/global_extensions/'
```

Comment:
did we now agree on `.pipeline/tmp` for "temp stuff"? Is it documented that piper users should gitignore that?